portfolio/src/main/java/com/google/sps/servlets/CommentsServlet.java

```diff
@@ -0,0 +1,111 @@
+// Copyright 2019 Google LLC
+//
+// Licensed under the Apache License, Version 2.0 (the "License");
+// you may not use this file except in compliance with the License.
+// You may obtain a copy of the License at
+//
+//     https://www.apache.org/licenses/LICENSE-2.0
+//
+// Unless required by applicable law or agreed to in writing, software
+// distributed under the License is distributed on an "AS IS" BASIS,
+// WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+// See the License for the specific language governing permissions and
+// limitations under the License.
+
+package com.google.sps.servlets;
+
+import java.io.IOException;
+import java.util.ArrayList;
+import java.util.List;
+import javax.servlet.annotation.WebServlet;
+import javax.servlet.http.HttpServlet;
+import javax.servlet.http.HttpServletRequest;
+import javax.servlet.http.HttpServletResponse;
+
+/** Servlet that handles comments data */
+@WebServlet("/comments")
+public class CommentsServlet extends HttpServlet {
+
+  private ArrayList<String> comments;
+  private ArrayList<String> names;
+  private ArrayList<ArrayList<String>> namesAndComments;
+  private ArrayList<String> labels;
+
+  
+  @Override
+  public void init() {
+    comments = new ArrayList<>();
+    names = new ArrayList<>();
+    namesAndComments = new ArrayList<ArrayList<String>>();
+    labels = new ArrayList<>();
+  }
+  
+  /* responds with a JSON string containing names and comments*/
+  @Override
+  public void doGet(HttpServletRequest request, HttpServletResponse response) throws IOException {
+    // Convert the ArrayLists to JSON
+    namesAndComments.add(names);
+    namesAndComments.add(comments);
+    labels.add("names");
+    labels.add("comments");
```

Comment:
Seems like these labels are always the same. Do we need to store them in a list or can we just hardcode them and use them when converting to json?